openmrs/omod/src/main/java/org/openmrs/projectbuendia/webservices/rest/OrderResource.java

```diff
@@ -341,6 +409,30 @@ private Order reviseOrder(Order order, SimpleObject edits) {
                     }
                     break;
 
+                case "start_millis":
```

Comment:
@capnfabs On Ping's initial commit for this functionality he removed the case "stop" from the switch. I kept it. Do we need it?